lib/plugins/aws/package/compile/events/eventBridge/index.js

```diff
@@ -87,6 +87,37 @@ class AwsCompileEventBridgeEvents {
           required: ['inputTemplate'],
           additionalProperties: false,
         },
+        retryPolicy: {
+          type: 'object',
+          properties: {
+            maximumEventAge: {
+              type: 'integer',
+              minimum: 60,
+              maximum: 86400,
+            },
+            maximumRetryAttempts: {
+              type: 'integer',
+              minimum: 0,
+              maximum: 185,
+            },
+          },
+        },
+        deadLetterConfig: {
+          type: 'object',
+          properties: {
+            'Fn::GetAtt': {
```

Comment:
What about other CF functions such as `Fn::Import`, `Sub` and similar? I believe we could support them as well here. do you see any limitations?